src/microbe_stage/MicrobeStage.cs

```diff
@@ -515,6 +524,9 @@ public void MoveToMulticellular()
             throw new Exception("failed to keep the current scene root");
         }
 
+        // Temporary
```

Comment:
"Temporary" this needs to go before merging or this needs a massive TODO slapped on this and an issue opened to make multicellular not "break" the unlock system.